config.json

```diff
@@ -1,42 +1,54 @@
 {
-    "web": {
-        "port": 3000,
-        "title": "BitCannon",
-        "torrentsPerPage": 100,
-        "defaultSearchSorting": "seeders",
-        "defaultSearchOrder": "desc"
-    },
-    "database": {
-        "mongodb": {
-            "enabled": true,
-            "host": "localhost",
-            "port": 27017,
-            "collection": "bitcannon"
-        }
+  "web": {
+    "port": 3000,
+    "title": "BitCannon",
+    "torrentsPerPage": 100,
+    "defaultSearchSorting": "seeders",
+    "defaultSearchOrder": "desc"
+  },
+  "database": {
+    "mongodb": {
+      "enabled": true,
+      "host": "localhost",
+      "port": 27017,
+      "collection": "bitcannon"
+    }
+  },
+  "torrents": {
+    "whitelist": {
+      "enabled": false,
+      "categories": []
     },
-    "session": {
-        "secret": "asdasdasdasd43jn5kj2n43k"
+    "blacklist": {
+      "enabled": false,
+      "categories": []
+    }
+  },
+  "session": {
+    "secret": ""
```

Comment:
I have no problem with that. If we're going to re-brand we'll need a new name (could just use something generic for now). We should also try to come up with a roadmap so we can see all of the things that need to be done and track issues. If @stephen304 doesn't have any objections it perhaps wouldn't be a bad idea to update the README to tell people development has halted and continued elsewhere.